build.gradle

```diff
@@ -39,7 +39,6 @@ plugins {
   id "io.swagger.core.v3.swagger-gradle-plugin" version "2.2.8"
 
   id "com.github.spotbugs" version '5.0.13' apply false
-  id 'org.gradle.test-retry' version '1.5.2' apply false
```

Comment:
The test-retry plugin is bundled with the Gradle Enterprise plugin.

Reply:
@clayburn  is apache Kafka using Gradle entreprise ?

Reply:
That's what this PR proposes, publishing build scans to the Gradle Enterprise instance at https://ge.apache.org. You can see the configuration in the `settings.gradle` portion of the PR.

Because the Gradle Enterprise Gradle Plugin bundles in the test-retry plugin, it is necessary to remove these lines. Otherwise test-retry will be loaded twice and fail. The retry functionality remains the same.

Reply:
What's the license for the gradle enterprise plugin?